enhancements/agent-installer/interactive-installation-using-the-agent-based-installer-for-disconnected-environments.md

```diff
@@ -0,0 +1,237 @@
+  
+* Post-installation cluster management and day-2 operations are out of scope
+for this enhancement.
+
+* Securing the communication between the Assisted Installer UI, Assisted
+Service, and client browsers will be addressed in future updates.
+
+* Adding UI support to the current ABI flow (starting with install-config and
+then switching to UI post-boot) .
+
+## Proposal
+
+This enhancement proposes extending the functionality of the Agent Based
+Installer to support a fully interactive, UI-based installation flow. The
+core of this proposal is to reuse and adapt the already existing Saas UI and
+tailor it for ABI use cases where a user can install no more than one cluster.
+The UI captures all necessary configuration details from the user, which are 
+then used by Assisted Service to orchestrate the cluster deployment. The UI
+can be used to start an installation from scratch or from a partially
+configured environment.
+
+The UI will be deployed on the rendezvous host as a container using a new image
+to be included in the release payload. A console message will provide users
+with the UI's URL. The UI will collect crucial information to configure the
+cluster and provide host verification to ensure the hardware inventory fulfill
+the cluster topology. It will also check network connectivity between the hosts.
+The workflow will also include a mechanism to select and bundle common
+operators, ensuring they are installed and configured alongside the cluster
+itself.
+
+### Workflow Description
+
+This workflow is activated when the sentinel file, /etc/assisted/interactive-ui,
+is present in the Agent Based Installer ignition.
+
+**cluster creator** is a human user responsible for deploying a cluster.
+
+**Assisted Installer UI** is UI used to configure, initiate, and monitor the
+cluster installation. It is hosted on the rendezvous host.
+
+**rendezvous host** is a host on the cluster that serves the Assisted Service and 
+Assisted installer UI. It also serves as the bootstrap node before it reboots
+during cluster installation to become a member of the cluster.
+
+**OCP console** is the installed cluster's UI.
+
+1. The cluster creator obtains or generates an ISO and boots it on target
+cluster hosts.
+
+2. The URL to the Assisted Installer UI running on a predefined port is
+displayed on the console.
+   
+3. On a separate machine the cluster creator opens up the browser to the
+Assisted Installer UI running on the rendezvous host.
+   
+4. The cluster creator begins configuring the cluster through the Assisted
+Installer UI, by providing all the required details.
+	
+5. If there are no configuration issues, the cluster creator moves to the 
+download credentials screen and downloads files containing the kubeconfig and
+kubeadmin password.
+	
+6. Then the Assisted Installer UI shows a confirmation screen showing the 
+configuration that was entered and a list of operators that will be enabled.
```

Comment:
The list of operators is just a future ennhancement? I.e. currently OVE contains a single set of operators (not customizable). Or are you referring here for supporting a list of operators to actually deploy post/during installation?